test/Models/AbsIttCatalogItemSpec.js

```diff
@@ -6,61 +6,6 @@ var Rectangle = require('terriajs-cesium/Source/Core/Rectangle');
 var Terria = require('../../lib/Models/Terria');
 var AbsIttCatalogItem = require('../../lib/Models/AbsIttCatalogItem');
 
-describe('AbsIttCatalogItem._generateAllCombinations', function() {
```

Comment:
Looks like you removed the tests, but forgot to remove the code that was tested here.  I'm assuming you meant to change it to use your new `arrayProduct`.